README.md

```diff
@@ -133,6 +144,20 @@ ros2 run fusion-engine-driver fusion_engine_ros_driver --ros-args -p connection_
 
 ### Connecting Over Serial
 
+In serial mode, it is possible to access the rtk correction. To do this you need to run the ntrip client node of ros2 in parallel.
+To connect to a ntrip server, you have to follow the instructions as follows:
+
+```
+git clone https://github.com/LORD-MicroStrain/ntrip_client
+cd ntrip_client
+git checkout ros2
+cd ..
+colcon build --packages-select ntrip_client
+source install/local_setup.bash
+ros2 launch ntrip_client ntrip_client_launch.py host:=[HOST] mountpoint:=[MOUNTPOINT]  username:=[YOUR USERNAME] password:=[YOUR PASSWORD]
```

Comment:
Can you describe what HOST/MOUNTPOIUNT/USERNAME/PASS are?
Is the HOST the ntrip server to connect to?
What about the MOUNTPOINT? etc...